content/en/flux/use-cases/azure.md

```diff
@@ -54,6 +71,7 @@ az aks create \
  --network-policy="calico" \
  --enable-managed-identity \
  --enable-pod-identity \
+ --enable-workload-identity \
```

Comment:
Do both flag work? I expect the command to fail. Better set only `--enable-workload-identity`